cpp/include/cudf/io/detail/json.hpp

```diff
@@ -51,4 +51,9 @@ void write_json(data_sink* sink,
                 json_writer_options const& options,
                 rmm::cuda_stream_view stream,
                 rmm::mr::device_memory_resource* mr);
+
+std::unique_ptr<rmm::device_uvector<char>> normalize_quotes(
+  const cudf::device_span<std::byte>& inbuf,
```

Comment:
East const please.

Reply:
Do you mean `cudf::device_span<std::byte> const& inbuf` instead of `const cudf::device_span<std::byte>& inbuf`?

Reply:
Yes.

Reply:
Style updated.